android/source/VideoLocker/src/org/edx/mobile/view/registration/RegistrationHtmlTextView.java

```diff
@@ -28,8 +31,11 @@ public RegistrationHtmlTextView(RegistrationFormField field, View view) {
         this.mErrorView = (TextView) view.findViewById(R.id.txt_input_error);
         this.mInstructionView = (TextView) view.findViewById(R.id.txt_input_instruction);
 
-        // display label as HTML
-        mInputView.setText(Html.fromHtml(mField.getLabel()));
+        // display label as HTML and text to be centered horizontally
+        mInputView.setMovementMethod(LinkMovementMethod.getInstance());
+        mInputView.setGravity(Gravity.CENTER_HORIZONTAL);
+        mInputView.setText(Html.fromHtml(
+                String.format("<a href=%s>%s</a>", mField.getAgreement().getLink(), mField.getAgreement().getText())));
```

Comment:
Does this need to be `` "<a href=\"%s\">%s</a>" `` so the link ends up in quotes?